atomics/T1078.004/T1078.004.yaml

```diff
@@ -89,37 +81,45 @@ atomic_tests:
     prereq_command: |
       terraform version
     get_prereq_command: |
-        echo Please install terraform.
+        echo "Please install terraform via https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli (URL accurate as of Nov. 15, 2024)."
   - description: |
-      Install-Module -Name Az
+      Check if Azure CLI and Azure Powershell are installed.
+
+      * Login to Azure CLI with "az login", and login to Azure Powershell with "Connect-AzAccount". Sessions are not shared.
+      * Azure Powershell used in this test as they have better automation performance and error logging than Azure CLI.
     prereq_command: |
       try {if (Get-InstalledModule -Name Az -ErrorAction SilentlyContinue) {exit 0} else {exit 1}} catch {exit 1}
     get_prereq_command: |
-      Install-Module -Name Az -Scope CurrentUser -Force
+      $ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindowsx64 -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
   - description: |
       Check if the user is logged into Azure.
     prereq_command: |
-      az account show
+      try {if (-not (Get-AzContext)) { exit 1 } else { exit 0 }} catch {exit 1}
     get_prereq_command: |
-      echo Configure your Azure account using: az login. 
+      echo "* Configure your Azure account using: Connect-AzAccount"
   - description: |
       Create dependency resources using terraform
+
+      * If fail to meet prereq, navigate to T1078.004-2 using "cd $PathToAtomicsFolder/T1078.004/src/T1078.004-2/"
+      * Open the "terraform.tfvars" file and fill in the variables with your desired values.
+      * Re-run -GetPrereqs
     prereq_command: |
       try {if (Test-Path "$PathToAtomicsFolder/T1078.004/src/T1078.004-2/terraform.tfstate" ){ exit 0 } else {exit 1}} catch {exit 1}
     get_prereq_command: |
+      echo "Navigating to: $PathToAtomicsFolder/T1078.004/src/T1078.004-2/"
       cd "$PathToAtomicsFolder/T1078.004/src/T1078.004-2/"
       terraform init
       terraform apply -auto-approve
   executor:
     command: |
```

Comment:
Login commands are now obsolete. Assume the login as been done in `-GetPrereqs` and the session exists.